### DISCUSSION TITLE
The code in tutorial web page has some error

### DISCUSSION THREAD

**Discussion by Knight-X** [halide/Halide#1731] [General]
In the page http://halide-lang.org/tutorials/tutorial_lesson_01_basics.html, there are some error in the code.

Halide::Image<int32_t> output = gradient.realize(800, 600);

Image would raise an error:
      Image is not a member of Halide.

It should be Halide::Buffer<int32_t> output = graident.realize(800, 600);

I can fix the error, could anyone tell where source code of the page is ?

**Comment by jrk** [comment]
Good catch, thanks. (I thought `Halide::Image` was supposed still supposed to exist as a proxy for `Buffer`— @abadams?)

The web site code lives at https://github.com/halide/halide.github.com.

**Comment by abadams** [comment]
The tutorials on the website match the last Halide binary release. They're
not intended to be current with trunk Halide. We update them when we do a
release.
